pixeltable/tests/conftest.py

```diff
@@ -23,14 +23,16 @@ def init_env(tmp_path_factory) -> None:
     from pixeltable.env import Env
     # set the relevant env vars for Client() to connect to the test db
 
-    shared_home = pathlib.Path(os.environ.get('PIXELTABLE_HOME', '~/.pixeltable')).expanduser()
+    shared_home = pathlib.Path(os.environ.get('PIXELTABLE_HOME', str(pathlib.Path.home() / '.pixeltable'))).expanduser()
```

Comment:
This shouldn't need `expanduser()` any more if you're obtaining the home dir some other way.